conf/app.src

```diff
@@ -1,5 +1,5 @@
-SOURCE_URL=https://github.com/ampache/ampache/archive/3.9.0.tar.gz
-SOURCE_SUM=1e523b52c6402693102dac183d7597d6e0aebf63580167f45661d0f15f57c072
+SOURCE_URL=https://github.com/ampache/ampache/archive/f9ea16e7fc7a36636180241dc79f139b92bc8ff0.tar.gz
```

Comment:
Why we don't use `https://github.com/ampache/ampache/releases/download/4.1.1/ampache-4.1.1_all.zip`

What is https://github.com/ampache/ampache/archive/f9ea16e7fc7a36636180241dc79f139b92bc8ff0.tar.gz ?
Not a tagged version sources ?

Reply:
because there is still the issue `dropbox/dropbox-sdk` in 4.1.1 so I decide to move to 4.1.1 +the removal of `dropbox/dropbox-sdk` as describe here: https://github.com/YunoHost-Apps/ampache_ynh/pull/60/commits/5f967aa1c24cc072401328abb146112cf193c33e

And https://github.com/ampache/ampache/commit/f9ea16e7fc7a36636180241dc79f139b92bc8ff0 is when upstream app removed `dropbox/dropbox-sdk`